CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,52 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of
+fostering an open and welcoming community, we pledge to respect all people who
+contribute through reporting issues, posting feature requests, updating
+documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free
```

Comment:
I'm not sure if lines 8 - 21 came from legal, but if not - run this by them. We don't want to open ourselves up for any damage with potential botched wording.

Reply:
We're using the code of conduct from an existing Rapid7 project.

Reply:
👍 I get that - but can you confirm THEY had legal check off on it?

Reply:
@aanand-r7 will be reviewing. Good point.

Reply:
Approval from @aanand-r7  after adding this: https://github.com/rapid7/insightconnect-plugins/pull/4/commits/cf4bf91aa4ab2bc6f6c33f2f09bd03764b82ae6b

<img width="275" alt="Screen Shot 2019-06-10 at 10 51 30 AM" src="https://user-images.githubusercontent.com/30870727/59207972-c091fd80-8b6d-11e9-80a5-b3dcf1a9a8a9.png">